src/components/Hello/index.tsx

```diff
@@ -23,6 +26,16 @@ const Hello = () => {
           gap: '14px',
         }}
       >
+        <button
+          onClick={() => {
+            i18n.changeLanguage('pl');
```

Comment:
Would be better to toggle the language from PL to EN and vice versa

Reply:
This is just an example - this file will be deleted.